ffaffurr-md-1step.py

```diff
@@ -15,8 +16,12 @@
 #)
 time_start=time.time()
 
+parser = argparse.ArgumentParser()
```

Comment:
```suggestion
parser = argparse.ArgumentParser(
    description="""Runs simulation using provided parameters, adding CT+POL if applicable"""   
    formatter_class=argparse.ArgumentDefaultsHelpFormatter
)
```
Provides a small description and adds the default values to help message.